assignment-client/src/entities/EntityTreeSendThread.cpp

```diff
@@ -285,7 +270,7 @@ void EntityTreeSendThread::startNewTraversal(const ViewFrustum& view, EntityTree
                 });
             });
         } else {
-            _traversal.setScanCallback([&](DiffTraversal::VisibleElement& next) {
+            _traversal.setScanCallback([=](DiffTraversal::VisibleElement& next) {
```

Comment:
According to the C++-11 docs the `this` object is always captured implicitly by reference `[&]` whenever it is used in the lambda AND a default capture mode is specified.  I'm changing these lambdas to copy by value `[=]` so that we can get `lodScaleFactor` and `viewPosition` just once instead of every iteration of the loop.